tests/test_sorted_set.nim

```diff
@@ -117,25 +117,28 @@ suite "SortedSet: Sorted list based on red-black tree":
   test "Delete items":
     var seen: seq[int]
     let sub7 = keyList.len div 7
-    for n in toSeq(countup(0,sub7)).concat(toSeq(countup(3*sub7,4*sub7))):
-      let
-        key = keyList[n]
-        canDeleteOk = (key notin seen)
+    if sl.len == 0:
```

Comment:
this doesn't really fix the issue: it skips the test where in reality, when running the one test by itself, it should still perform the same test as before: the way to do this is to move the initialization code that sets up the test into a function and call it from both delete items and the other test